selfie.c

```diff
@@ -4519,6 +4517,29 @@ void emitStart() {
   // fixup jump at address 0 to here
   fixup_relative_JFormat(0, binaryLength);
 
+  // initialize s1 with the current bump pointer
+  emitADDI(REG_A0, REG_ZR, 0);
+  emitADDI(REG_A7, REG_ZR, SYSCALL_BRK);
+
+  emitECALL();
+
+  talloc();
+
+  // align the current bump pointer for double word access
+  emitADDI(REG_A0, REG_A0, SIZEOFUINT64 - 1);
```

Comment:
This block repeats below. Can you write a procedure `emitAlign` or something similar to `emitLeftShiftBy`?

Reply:
Also, add a comment with the formula that is computed here.

Reply:
I introduced a function called `emitRoundUp`, because that's really the semantic. 7aa2fa51adf23408a9b3d35480e74fb720d706a8 and 34ec8fe1ccf452fbad23e7e338d5916324193831